src/main/scala/amailp/intellij/robot/psi/reference/ResourceValueReference.scala

```diff
@@ -8,6 +8,8 @@ import com.intellij.openapi.vfs.VirtualFile
 import com.intellij.psi.{PsiElement, PsiReferenceBase}
 
 class ResourceValueReference(element: ResourceValue) extends PsiReferenceBase[ResourceValue](element) with ExtRobotPsiUtils {
+  val resourceFilePath: String = getElement.getText.replace("${/}", "/")
```

Comment:
```suggestion
  val resourceFilePath: String = getElement.getText.replace("${/}", java.io.File.separator)
```
In order to work also on Windows OS

Reply:
I did just that at first, but that didn't work because findFileByRelativePath inside expects a "/". I checked it on windows. Maybe I'm missing something